### PR TITLE
refactor: clean up stepper component code and improve formatting

### PR DESCRIPTION
**Pull request by ScreenTechnicals**

### CODE REVIEW

**Comment by meness** [comment]
test

**Comment by meness** [comment]
@dr-github-pro how are u doing?

File: src/lib/stepper.component.tsx [hunk 1]
```diff
@@ -1,39 +1,36 @@
...
-      className
-    )}
-  />
-);
+//       "w-5 md:min-w-20 h-px bg-gradient-to-r dark:from-white/50 dark:via-white/50 dark:to-white/50 light:from-black/50 light:via-black/50 light:to-black/50",
+//       orientation === "vertical" && "my-2 w-px md:min-w-px md:min-h-20 h-5",
+//       className
+//     )}
+//   />
+// );
```
[the remark this answers is not in the document]
Comment (meness) [reply src/lib/stepper.component.tsx:15]: @dr-github-pro hey

**Comment by meness** [comment]
@dr-github-ai hey

**Comment by ScreenTechnicals** [comment]
---------hello--------

**Comment by meness** [comment]
fuck

**Comment by meness** [comment]
hello

**Comment by ScreenTechnicals** [comment]
-----ok-done----

**Comment by ScreenTechnicals** [comment]
-------

**Comment by meness** [comment]
fuck

**Comment by meness** [comment]
another time

**Comment by meness** [comment]
fdfdfdf

**Pull request closed without merging**